controller/backup_volume_controller.go

```diff
@@ -220,11 +220,6 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 	// Examine DeletionTimestamp to determine if object is under deletion
 	if !backupVolume.DeletionTimestamp.IsZero() {
 
-		if unused, err := bvc.isBackupNotBeingUsedForVolumeRestore(backupVolumeName); !unused {
```

Comment:
Could you please explain why we can remove this check but still ensure that the backup is not deleted while there is volume using it for restoration? @ChanYiLin

Reply:
because below L#228
```
Delete the backup volume from the remote backup target
```
https://github.com/longhorn/longhorn-manager/pull/1893/files#diff-4967e3520dbda6bc373b2fea83e911dd65aac0ab4f48a53f977413620ce4ba7dR228

This will fail because of the lock of the file in the remote backup store
so even we don't have this check
the CR wouldn't be delete when the file is still used for restoration

Reply:
because below L#228
```
Delete the backup volume from the remote backup target
```
https://github.com/longhorn/longhorn-manager/pull/1893/files#diff-4967e3520dbda6bc373b2fea83e911dd65aac0ab4f48a53f977413620ce4ba7dR228

This will fail because of the lock of the file in the remote backup store
so even we don't have this check
the CR wouldn't be delete when the file is still used for restoration